duckscript/src/lib.rs

```diff
@@ -59,7 +59,6 @@
     order_dependent_trait_objects,
     overflowing_literals,
     overlapping_range_endpoints,
-    panic_fmt,
```

Comment:
why remove it?

Reply:
With stable rustc 1.53.0 I receive "warning: unknown lint: `panic_fmt`"

Reply:
ya, i have a script to update to latest lints after i upgrarde my local machine. but its not related to this pr.